src/components/UserInfo.js

```diff
@@ -14,4 +15,8 @@ export default class UserInfo {
     this._nameElement.textContent = userName;
     this._descriptionElement.textContent = userDescription;
   }
+
+  setAvatar(link) {
```

Comment:
It's great to have a separate setter for the avatar, since it's controlled by a different form